src/test/java/com/loadero/models/LoaderoTest.java

```diff
@@ -228,4 +228,11 @@ public void testReadAll() throws IOException {
         List<com.loadero.model.Test> tests = com.loadero.model.Test.readAll();
         Assertions.assertNotNull(tests);
     }
+
+    @Test
+    @DisabledIfEnvironmentVariable(named = "LOADERO_BASE_URL", matches = ".*localhost.*")
+    public void testReadAllRuns() throws IOException {
```

Comment:
this should be in its own test file, as it has a separate model. TestRun doesn't seem to have any tests at all

Reply:
TestRun is tested in the TestPolling file, because it contains polling logic. I guess .readAll() can be placed in the same file.

Reply:
that would make more sense imo. but we might want to think about a file dedicated to TestRun testing in the future

Reply:
Well, we can rename `TestPolling` into `TestRunTest`(?) as currently `TestPolling` mainly contains tests that executes logic inside `TestRun`, because this is where `.poll()` method is located and other `TestRun`'s methods.

Reply:
let's keep it as-is for now, once we have more tests there then we can do that